server/main.cpp

```diff
@@ -36,12 +35,8 @@ int main(int argc, char *argv[]) {
     if (log_to_file) {
         spdlog::set_default_logger(spdlog::daily_logger_mt("query-pdb", "server.log"));
     }
-
-    downloader storage(download_path, download_server);
```

Comment:
creating only one instance of `downloader` makes sense because we need to prevent concurrent calls to the `download_impl` function